### ISSUE TITLE
ctpu up failed. Maybe the recently code .

### ISSUE THREAD

**Issue by HisiFish** [tensorflow/tpu#165]
In the past several days, I used 'ctpu up' to create VM & TPU。
It worked well.

But I tried many times today, it always failed.
This is the output:

```
aaflier@cloudshell:~ (ck-augment-208602)$ ctpu up
ctpu will use the following configuration:

  Name:                 aaflier
  Zone:                 us-central1-b
  GCP Project:          ck-augment-208602
  TensorFlow Version:   1.9
  VM:
      Machine Type:     n1-standard-2
      Disk Size:        250 GB
      Preemptible:      false
  Cloud TPU:
      Size:             v2-8
      Preemptible:      false

OK to create your Cloud TPU resources with the above configuration? [Yn]:                                         Y
2018/08/01 16:20:02 Creating Compute Engine VM aaflier (this may take a minute)...
2018/08/01 16:20:02 Creating TPU aaflier (this may take a few minutes)...
2018/08/01 16:20:09 Created Compute Engine VM aaflier!
2018/08/01 16:20:10 TPU operation still running...
2018/08/01 16:20:32 TPU operation still running...
2018/08/01 16:20:54 TPU operation still running...
2018/08/01 16:21:17 TPU operation still running...
2018/08/01 16:21:39 TPU operation still running...
2018/08/01 16:22:01 TPU operation still running...
2018/08/01 16:22:24 TPU operation still running...
2018/08/01 16:22:46 TPU operation still running...
2018/08/01 16:23:08 TPU operation still running...
2018/08/01 16:23:30 TPU operation still running...
2018/08/01 16:23:52 TPU operation still running...
2018/08/01 16:24:13 Created TPU aaflier!
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x6677b3]
goroutine 1 [running]:
github.com/tensorflow/tpu/tools/ctpu/ctrl.(*ResourceManagementCP).IsProjectInGoogleOrg(0xc4200511b0, 0x773340, 0xc420052dc0, 0xc42000c798)
        /tmp/ctpu-release/src/github.com/tensorflow/tpu/tools/ctpu/ctrl/resourcemgmt.go:119 +0xe3
github.com/tensorflow/tpu/tools/ctpu/commands.(*upCmd).Execute(0xc4200795e0, 0x773380, 0xc4200160f0, 0xc4200568a0, 0x0, 0x0, 0x0, 0x6e05a0)
        /tmp/ctpu-release/src/github.com/tensorflow/tpu/tools/ctpu/commands/up.go:449 +0x2c3
github.com/google/subcommands.(*Commander).Execute(0xc420070000, 0x773380, 0xc4200160f0, 0x0, 0x0, 0x0, 0x5)
        /tmp/ctpu-release/src/github.com/google/subcommands/subcommands.go:141 +0x29f
github.com/google/subcommands.Execute(0x773380, 0xc4200160f0, 0x0, 0x0, 0x0, 0xc420052700)
        /tmp/ctpu-release/src/github.com/google/subcommands/subcommands.go:385 +0x5f
main.main()
        /tmp/ctpu-release/src/github.com/tensorflow/tpu/tools/ctpu/main.go:87 +0xd5e
aaflier@cloudshell:~ (ck-augment-208602)$ ctpu delete
```

This error, which would be the golang equivalent of a null pointer exception, appears to be thrown from this[1] goroutine which checks whether your project belongs to the google.com organization (there are some different types of restrictions for such projects). I can see that your project does not belong to *any* organization, which could explain the null pointer. Also, considering that this piece of code was added recently (the corresponding merge seems to be from two days ago[2]), it would make sense that you didn't face this problem earlier.

If that is the case indeed, this is an issue on our side. However, I would like to communicate this to my colleagues to confirm my thoughts. I will follow up with you later within the day.

[1] https://github.com/tensorflow/tpu/blob/master/tools/ctpu/ctrl/resourcemgmt.go#L111
[2] https://github.com/tensorflow/tpu/commit/9f66664f055e7943d157ba94b7bd5b0bc5b3d54e#diff-44649e4ff88feeb0adeb5df4b70076ee

**Comment by saeta** [comment]
Hi @HisiFish Thank you for this bug report! You're absolutely right that I goofed up here. I've got a fix in the works, and will release it ASAP. I'll (try to remember to) update this bug when I get the next version of `ctpu` released.

All the best!
-Brennan

**Comment by saeta** [comment]
@HisiFish I've now released `ctpu` 1.5, so it's available for download from `dl.google.com`; you can grab a copy of the binary now (see below). Google Cloud Shell will auto-update to the latest version over the next week or two.

Let me know if you encounter any new issues!

All the best,
-Brennan

```
saeta@saeta0:/tmp$ mkdir foobar
saeta@saeta0:/tmp$ cd foobar/
saeta@saeta0:/tmp/foobar$ wget https://dl.google.com/cloud_tpu/ctpu/latest/linux/ctpu
--2018-08-02 13:24:56--  https://dl.google.com/cloud_tpu/ctpu/latest/linux/ctpu
Resolving dl.google.com (dl.google.com)... 2607:f8b0:4005:804::200e, 216.58.194.174
Connecting to dl.google.com (dl.google.com)|2607:f8b0:4005:804::200e|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 8016209 (7.6M) [application/octet-stream]
Saving to: ‘ctpu’

ctpu                100%[===================>]   7.64M  37.3MB/s    in 0.2s    

2018-08-02 13:24:57 (37.3 MB/s) - ‘ctpu’ saved [8016209/8016209]

saeta@saeta0:/tmp/foobar$ chmod a+x ctpu
saeta@saeta0:/tmp/foobar$ ./ctpu version
ctpu version: 1.5
saeta@saeta0:/tmp/foobar$ 
```
